### PR TITLE
removed some _G entries

### PR DESCRIPTION
**Pull request by Valansch**

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
-_DEBUG = false
+_DEBUG = true
```
Comment (iltar) [inline config.lua]: I don't think this should be committed

File: utils/game.lua [hunk 2]
```diff
@@ -39,4 +39,12 @@ function Game.get_player_by_index(index)
     end
 end
 
+function Game.player_print(str)
+    if game.player then
```
Comment (iltar) [inline utils/game.lua]: > The player typing at the console - nil in all other instances. See `LuaGameScript::players` for accessing all players. 

https://lua-api.factorio.com/latest/LuaGameScript.html#LuaGameScript.player

Comment (Valansch) [reply utils/game.lua]: This is the desired behavior.

**Comment by grilledham** [comment]
I haven't tested it, but assuming nothing breaks I approve.

**Pull request merged**